packages/react/src/menu-content-base/MenuContentBase.tsx

```diff
@@ -1,20 +1,28 @@
 import * as RadixMenu from "@radix-ui/react-dropdown-menu";
-import { forwardRef } from "react";
+import { type ElementType, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
 
 import { AnimatePresence } from "../animate-presence";
 import { Box, type BoxProps } from "../box";
 import { Transition } from "../transition";
 import * as styles from "./MenuContentBase.css";
 
-type MenuContentProps = BoxProps<
-  "div",
-  {
-    open: boolean | undefined;
-  }
+export type MenuContentBaseProps<
+  T extends ElementType = "div",
+  P = unknown,
+> = BoxProps<
+  T,
+  ExtendProps<
+    {
+      open: boolean | undefined;
+    } & styles.ContentVariants,
+    P
+  >
 >;
 
-export const MenuContentBase = forwardRef<HTMLDivElement, MenuContentProps>(
-  ({ children, className, open, ...props }, ref) => {
+export const MenuContentBase = forwardRef<HTMLDivElement, MenuContentBaseProps>(
+  ({ children, className, minW, open, ...props }, ref) => {
```

Comment:
Don't we need to set a default value for it vai?

Reply:
no - by default it'll be ignored and it won't set any `minWidth` style at all

but if you want you can set it to `trigger` to match the trigger width - or once you've already done that you can reset it back by overriding it with `0`